datasets/doc/source/index.rst

```diff
@@ -67,6 +74,19 @@ Thanks to using Hugging Face's ``datasets`` used under the hood, Flower Datasets
 - Jax
 - Arrow
 
+Here are a few of the ``Partitioner`` s that are available: (for a full list see `link <https://flower.ai/docs/datasets/ref-api/flwr_datasets.partitioner.html#module-flwr_datasets.partitioner>`_ )
```

Comment:
```suggestion
Here are a few of the ``Partitioner`` s that are available: (for a full list see `link <ref-api/flwr_datasets.partitioner.html#module-flwr_datasets.partitioner>`_ )
```

Reply:
In the framework docs we use most of the times (or always?) relative links like the one I suggest. I believe it's better for caching etc.

Reply:
Yes, thanks. That makes more sense.